google-http-client-apache-legacy/pom.xml

```diff
@@ -7,9 +7,9 @@
     <version>1.27.1-SNAPSHOT</version><!-- {x-version-update:google-http-client-parent:current} -->
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <artifactId>google-http-client-apache-legacy</artifactId>
+  <artifactId>google-http-client-apache</artifactId>
```

Comment:
why remove the "-legacy"?

Reply:
We haven't yet published either artifact yet. Originally, I had planned for the new version to be backwards compatible, so we would keep 2 separate artifacts at the same version. However, after it became clear that we needed to have breaking changes to support the "new" Apache HttpClient implementations, it made more sense to have the same artifact with 2 major versions.

This is totally up for discussion though.